rental.rb

```diff
@@ -0,0 +1,20 @@
+require 'time'
+require './book'
+require './person'
+
+class Rental
+  attr_reader :person, :book
+  attr_accessor :date
```

Comment:
- [ ] The `person` and `book` should be set to getter and setter - attr_accessor - as well

Reply:
@julie-ify, I respectfully disagree here, every time a rental object is instantiated, it has both the renter and the rented book instances. It doesn't make any sense to come back and alter those values.